### ISSUE TITLE
training 

### ISSUE THREAD

**Issue by opslash** [devgeek2700/Sign-Language-recognition-using-openCV#1]
1. I want to know how you label the dataset.
2. which type of algorithm do you have used?
3. from where have you learned about this project?
4. I want a training dataset file that you use in this project.

**Comment by devgeek2700** [comment]
first solution
first I have made all the letters/numbers folder manually in Image folder then in line 12 I have given the path and the letter name in which the clicked images should be saved, then from line 15 to line 50 I have written the code to open the webcam and detect the hands after clicking on letter 'a' all the images are storing the specific path.

**Comment by devgeek2700** [comment]
second solution
algorithm is Convolutional Neural Network (CNN)  to train our model and identify the images and for training I had use online Teachable Machine.
link is :- https://teachablemachine.withgoogle.com

**Comment by devgeek2700** [comment]
third solution 
I have learn this project from Github and topics from Youtube.

**Comment by devgeek2700** [comment]
fourth solution 
All the pretrained file in present in the model folder.
